extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/common/TaskLaneConfig.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.k8s.overlord.common;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class TaskLaneConfig
+{
+  private final String label;
+  private final double capacityRatio;
+  private final String policy;
+
+  @JsonCreator
+  public TaskLaneConfig(
+      @JsonProperty("label") String label,
```

Comment:
The label and config structure feels a little unintuitive. `label` here is actually a comma-separated string of labels. Also the labels themselves are not easy to figure out since they're dynamically generated based on the content of the spec. We should think of a way to make these labels (if we want to keep them at all) be user discoverable.

Reply:
For non-MSQ tasks configuration since label is same as task type I assume user already know that info.
For MSQ tasks configuration, one way is to add more document about how different labels are used in MSQ task so user can find out that but will think about whether there is a better way to do that.